### ISSUE TITLE
Incorrect pins in manual for SKR MINI E3 V3.0?

### ISSUE THREAD

**Issue by hrafnfloki** [bigtreetech/MicroProbe#31]
I'm attempting to follow the instructions of the MicroProbe V2 manual, which is on this GitHub, and I've been running into error after error.

>note: this is the location of the previous definition
>  1325 | #define Z_MIN_PROBE_PIN PC13 // The detection IO on the SKR 3 is PC13

>error: 'PE5' was not declared in this scope; did you mean 'PE_5'?

I followed the manual word for word.
So I'm not sure if the manual is out of date, because it was last updated earlier this year? Does anyone know if I'm doing something wrong and what the correct "control and detection I/Os" are?

**Comment by hrafnfloki** [comment]
![image](https://github.com/bigtreetech/MicroProbe/assets/80012071/ef20e855-7987-4c18-8017-916d7287ea33)

**Comment by guiabc38** [comment]
I have the same problem that you

**Comment by forplay0709** [comment]
Yes they are incorrect. I was able to get it to work only if Z_MIN_PROBE_PIN=PC14 and if PROBE_ENABLE_DISABLE=PA1. There's not even a pin described as PE5 on the motherboard. I don't know who wrote the manual and when..

**Comment by vitung8** [comment]
> Yes they are incorrect. I was able to get it to work only if Z_MIN_PROBE_PIN=PC14 and if PROBE_ENABLE_DISABLE=PA1. There's not even a pin described as PE5 on the motherboard. I don't know who wrote the manual and when..

Can you share the firmware?
